pkgs/servers/nfs-ganesha/default.nix

```
@@ -43,16 +44,25 @@ stdenv.mkDerivation rec {
     nfs-utils
   ];
 
+  postPatch = ''
+    substituteInPlace src/tools/mount.9P --replace "/bin/mount" "/usr/bin/env mount"
+  '';
+
   postFixup = ''
     patchelf --add-rpath $out/lib $out/bin/ganesha.nfsd
   '';
 
+  postInstall = ''
+    install -Dm755 $src/src/tools/mount.9P $tools/sbin/mount.9P
```

```suggestion
    install -Dm755 $src/src/tools/mount.9P $tools/bin/mount.9P
```
nix map `sbin`  directly back to bin. Better install it directly into `bin`